app/src/main/java/com/example/oddhours/ui/addjob/AddJobFragment.kt

```diff
@@ -39,6 +39,8 @@ class AddJobFragment : Fragment() {
     @SuppressLint("DefaultLocale")
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        val locationPattern = Regex("[0-9]|[^A-Za-z, ]")
+        val companyPattern = Regex("[^A-Za-z0-9,& ]")
```

Comment:
not sure what the location pattern is allowing / denying but i can't enter an address of "249 yonge street" it won't let me
![image](https://user-images.githubusercontent.com/11465548/180889445-45772375-3d46-4f07-be1d-01f9f557e2c6.png)

also if i enter something like: 'rw&co - cashier' for company name it doesn't like it
![image](https://user-images.githubusercontent.com/11465548/180889672-1b48a62b-f2d5-437c-bd8b-ad56d3f35b74.png)

i know we want to prevent sql injection by preventing inputs with symbols but maybe that should be a backend solution?

Reply:
those are good catches. I'll update the regex, for location my assumption was we only allow in the form of "Toronto, Ontario" not exact location. Do we need to worry about SQL injection as of now? considering that there is no cloud component and all of the data is offline on user's device.

Reply:
right. since we don't even have to worry about that i think it'll be better to just have characters we're excluding from location instead. what do you think?

Reply:
yaa and for company name we can still have the current validation as it is right?

Reply:
i didn't check the new validation for company name. did you update it? can it allow a company name with special characters now?